app/frontend/components/pages/nufiPreview/lists.tsx

```diff
@@ -0,0 +1,170 @@
+  'In-app NFT gallery, Cardano DEX, trading and more',
+]
+
+export const nufiComparisonContent = [
+  {
+    label: 'Non-custodial storage and staking',
+    adalite: true,
+    nufi: true,
+  },
+  {
+    label: 'Ledger and Trezor compatible',
+    adalite: true,
+    nufi: true,
+  },
+  {
+    label: 'Chrome extension + dApp connector to connect to NFT marketplaces, DEXes, DeFi and more',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'Visual NFT gallery',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'Integrated Cardano DEX',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'Send multiple assets in 1 transaction',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'See fiat values of Cardano tokens',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'Export staking rewards/transaction history',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'Multiple blockchains supported',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'Buy/sell crypto with card or bank transfer',
+    adalite: false,
+    nufi: true,
+  },
+  {
+    label: 'In-app cross-chain trading',
+    adalite: false,
+    nufi: true,
+  },
+] as const
+
+const howToUseNufiLinks = [
+  {
+    label: (
+      <Fragment>
+        Install Chrome Extension <br /> + dApp Connector
+      </Fragment>
+    ),
+    additionalText:
+      '(Recommended. Select this option to connect to dApps to buy/sell NFTs, trade tokens, discover DeFi and more)',
+    Icon: ExtensionIcon,
+    href: 'https://chrome.google.com/webstore/detail/nufi/gpnihlnnodeiiaakbikldcihojploeca',
+  },
+  {
+    label: 'Open Web Wallet',
+    additionalText: '(Compatible with all browsers. No dApp connector)',
+    Icon: WebIcon,
+    href: 'https://wallet.nu.fi',
+  },
+]
+const afterImporting = {
+  title: 'After you import your wallet into NuFi:',
+  list: [
+    'You’ll see the same accounts and assets you see on AdaLite',
+    'Your staking setup and rewards will be the same with no further action needed',
+    'Your wallet’s receiving address will be the same',
+    'You’ll use a password to log in instead of entering your seed phrase or connecting a device',
+  ],
+}
+
+export const importSection = [
+  {
+    title: 'Choose how you’d like to use NuFi',
+    content: howToUseNufiLinks.map(({href, label, additionalText, Icon}) => (
+      <div style={{width: '85%'}} key={href}>
+        <NufiCard
+          href={href}
+          text={
+            <Fragment>
+              {label} <br />
+            </Fragment>
+          }
+          icon={<Icon />}
+        />
+        <p style={{paddingTop: 5}}>{additionalText}</p>
+        <br />
+      </div>
+    )),
+  },
+  {
+    title: 'Import your AdaLite wallet:',
+    content: (
+      <Fragment>
+        <div>
+          For mnemonic/seed phrase wallets:
+          <br />
+          Click 'Restore Wallet' and enter your 12, 15 or 24-word seed phrase.
```

Comment:
```suggestion
          Click 'Restore Wallet' and enter your 15 or 24-word seed phrase.
```
technically, 12-word mnemonics in Adalite work differently than in Nufi so I wouldn't advertise them. 12-word mnemonics in Adalite resolve to legacy Daedalus wallets, whereas NuFi derives "shelley" wallets the same way as e.g. Trezor or Typhon wallet do. You can discuss this with Michal Petro as I'd be out next week - he should have the context as well